src/main/java/net/datafaker/providers/base/TimeAndDate.java

```diff
@@ -21,8 +21,8 @@
  */
 public class TimeAndDate extends AbstractProvider<BaseProviders> {
 
-    private static final int DEFAULT_MIN_AGE = 18;
-    private static final int DEFAULT_MAX_AGE = 65;
+    public static final int DEFAULT_MIN_AGE = 18;
```

Comment:
Probably package protected would be fine too?

Reply:
Sure I'll adjust in another in the next few days